documentation/book/getting-started.adoc

```diff
@@ -322,3 +322,13 @@ strimzi.io/kind: topic
 NOTE: When the Topic Operator is deployed manually the `strimzi.io/cluster` label is not necessary.
 
 The topic ConfigMap contains the topic configuration in a specific format. The ConfigMap format is described in <<topic_config_map_details>>.
+
+==== Logging
+The `logging` field allows the configuration of loggers. These loggers are listed below.
+[source]
+property.topic-operator.root.logger
```

Comment:
The property is not really an logger, or? It was just used to configure the logger from the env. variable, or? So I think mentioning the `rootLogger.level` should be IMHO enough.